perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/ExtendMembershipException.java

```diff
@@ -4,7 +4,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Checked version of ExtendMembershipException.
+ * Thrown when the membership could not be extended for various reasons which is sent to the constructor of the exception
```

Comment:
If we really want to inform about reason in the constructor, I would do that in separate sentence.
`The reason is sent as a parameter of the exception.`